config/install/user.role.authenticated.yml

```diff
@@ -0,0 +1,40 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.wysiwyg
+  module:
+    - filter
+    - fontawesome
+    - media
+    - system
+id: authenticated
+label: 'Authenticated user'
+weight: 1
+is_admin: false
+permissions:
+  - 'access content'
+  - 'access fontawesome additional settings'
+  - 'access media overview'
+  - 'administer media'
+  - 'create document media'
+  - 'create image media'
+  - 'create media'
+  - 'create remote_video media'
+  - 'delete media'
+  - 'delete own document media'
+  - 'delete own image media'
+  - 'delete own remote_video media'
+  - 'edit any document media'
```

Comment:
I can see we might need to do something for editing media that is attached to content that is attached to a group that isn't _owned_ by the user, but again not sure this is the way or needs to be done now.